test/browser/features/international-address-enter-address.feature

```diff
@@ -145,24 +145,13 @@ Feature: International address - Enter your address
     Then they see an error summary with failed validation message: "Region must only include letters a to z, numbers 0 to 9 and .,-\/* or '"
     Then they see an error summary with failed validation message: "Town, suburb or city must only include letters a to z, numbers 0 to 9 and .,-\/* or '"
 
-  Scenario: Change selected international country, selected same country leads to international page
-    Given they have selected the country "Kenya"
-    When they click the continue button
-    Then they should see international address form
-    And they see the change country link "Kenya"
-    When they click change country link
-    Then they should see the country selector page
-    When they click the continue button
-    Then they should see international address form
-
   Scenario: Change selected international country, selected different international country leads to international address page
     Given they have selected the country "Kenya"
     When they click the continue button
     Then they should see international address form
     And they see the change country link "Kenya"
     When they click change country link
     Then they should see the country selector page
-    And they should also see the selected country is still "Kenya"
```

Comment:
so this should be explicitly changed, to show what the new behaviour is, I suppose I would have preferred

`And they should also see that no country is selected` or  `And they should also see the selected country is ""`